sdcm/sct_config.py

```diff
@@ -1562,6 +1562,12 @@ class SCTConfiguration(dict):
 
         dict(name="run_scylla_doctor", env="SCT_RUN_SCYLLA_DOCTOR", type=boolean,
              help="Run scylla-doctor in artifact tests"),
+
+        dict(name="enable_force_gossip_topology_changes_on_upgrade",
```

Comment:
let's make it `force_gossip_topology_changes` so it's not upgrade specific only - adjust `BaseScyllaCluster.parallel_node_operations` to use this setting too.

Reply:
I don't think so.

This is for configuration to happen at a very specific place 

In your case of bootstrapping, you should be checking the existence of a feature, not the many configuration options that might affect it

Reply:
Still can be used at specific place in upgrade procedure and additionally used to disable raft in typical longevity tests - why limiting only to upgrade?

Reply:
How I want the cluster to be configured at creation, and what action one wants to todo at upgrade, are not always the same.

You would be put a limitation about what variations we could test in an upgrade test.

I do think we should have something general like scylla_yaml_upgrade for it to be future proof and not invented again and again, for raft, then topology, then tablets.

But it's need to distinct from fresh cluster configuration

Reply:
ok, I'm not insisting. 
In that case let it be this way and we need to add one condition to `parallel_node_operations` - when scylla.yaml is having `force_gossip_topology_changes` then it should be set to `False`. WDYT?

Reply:
Yeah this is something we need todo regardless of this PR

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/7827